src/engine_event_handler.py

```diff
@@ -24,48 +30,83 @@ def __use_next_figure(self) -> None:
     def __is_game_end(self) -> bool:
         return not self._board.is_cells_empty(self.__current_figure.points)
 
+    def __is_needed_to_move_figure_down(self) -> bool:
+        if self.__tick_count < self.__ticks_for_move_down:
+            return False
+
+        self.__tick_count = 0
+
+        return True
+
+    def __decrease_ticks_for_move_down(self, removed_lines: int) -> None:
+        self.__ticks_for_move_down -= removed_lines
+        self.__ticks_for_move_down = max(self.__ticks_for_move_down, 1)
+
+    def __handle_figure_move_right(self) -> None:
+        new_points, new_position = FigureMover.move_right(self.__current_figure)
+
+        if self._board.is_cells_empty(new_points):
+            self.__current_figure.position = new_position
+
+    def __handle_figure_move_left(self) -> None:
+        new_points, new_position = FigureMover.move_left(self.__current_figure)
+
+        if self._board.is_cells_empty(new_points):
+            self.__current_figure.position = new_position
+
+    def __handle_figure_move_down(self) -> None:
+        new_points, new_position = FigureMover.move_down(self.__current_figure)
+
+        if self._board.is_cells_empty(new_points):
+            self.__current_figure.position = new_position
```

Comment:
Looks like duplicates. Do you like my previous variant with `event->moving_func` mapping?

Reply:
Yes, for sure -- I gonna implement your approach, any concerns?